rclcpp/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Do you think downstream packages need this too?

Reply:
I wouldn't expect downstream packages to need it, but I suppose it's worth trying.